src/lang-promql/complete/hybrid.ts

```diff
@@ -250,7 +259,11 @@ export function analyzeCompletion(state: EditorState, node: SyntaxNode): Context
       if (!parent) {
         // this case is normally impossible since by definition, the identifier has 3 parents,
         // and in Lexer, there is always a default parent in top of everything.
-        result.push({ kind: ContextKind.MetricName }, { kind: ContextKind.Function }, { kind: ContextKind.Aggregation });
+        result.push(
+          { kind: ContextKind.MetricName, metricName: getMetricIdentifierValue(node, state) },
```

Comment:
In the grammar here the node is `Identifier`, so it is really weird that you need to get the node `MetricIdentifier` to get the value. I think you can do it directly with the node `Identifier`.

Reply:
so you can simply do `state.sliceDoc(node.from, node.to);`